src/test/shell/bazel/android/resource_processing_integration_test.sh

```diff
@@ -123,7 +123,7 @@ function test_persistent_multiplex_resource_processor() {
   setup_font_resources
 
   assert_build //java/bazel:bin --persistent_android_resource_processor \
-    --experimental_persistent_multiplex_busybox_tools
+    --persistent_multiplex_android_resource_processor
```

Comment:
With this approach we also wouldn't need to add a unique test case just for `DexBuilder` multiplex workers as it would be covered by the `persistent_multiplex_android_resource_processor` expansion here.

Reply:
we can consolidate in a followup change